share/locale/en.json

```diff
@@ -1,184 +1,184 @@
 {
-    "config_onion_service": "Configuring onion service on port {0:d}.",
-    "preparing_files": "Preparing files to share.",
-    "give_this_url": "Give this address to the person you're sending the file to:",
-    "give_this_url_stealth": "Give this address and HidServAuth line to the person you're sending the file to:",
-    "give_this_url_receive": "Give this address to the people sending you files:",
-    "give_this_url_receive_stealth": "Give this address and HidServAuth line to the people sending you files:",
+    "config_onion_service": "Setting up onion service on port {0:d}.",
+    "preparing_files": "Preparing share.",
+    "give_this_url": "Give this address to the recipient:",
+    "give_this_url_stealth": "Give this inbox address and HidServAuth line to the recipient:",
+    "give_this_url_receive": "Give this inbox address to the sender:",
+    "give_this_url_receive_stealth": "Give this inbox address and HidServAuth to the sender:",
     "ctrlc_to_stop":  "Press Ctrl+C to stop the server",
     "not_a_file": "{0:s} is not a valid file.",
     "not_a_readable_file": "{0:s} is not a readable file.",
-    "no_available_port": "Could not start the Onion service as there was no available port.",
-    "other_page_loaded": "Address loaded",
-    "close_on_timeout": "Stopped because timer expired",
-    "closing_automatically": "Stopped because download finished",
-    "timeout_download_still_running": "Waiting for download to complete",
-    "large_filesize": "Warning: Sending large files could take hours",
+    "no_available_port": "Could not find an available port to start the onion service",
+    "other_page_loaded": "Inbox address loaded",
+    "close_on_timeout": "Stopped because timeframe ran out",
+    "closing_automatically": "Stopped because you have the complete share",
+    "timeout_download_still_running": "Awaiting completion of share transfer to you",
+    "large_filesize": "Warning: Sending a large share could take hours",
     "systray_menu_exit": "Quit",
-    "systray_download_started_title": "OnionShare Download Started",
```

Comment:
In my head a download is something sent to me. This lacks any distinction to that effect.